docs/index.html

```diff
@@ -0,0 +1,95 @@
+  <![endif]-->
+</head>
+
+<body>
+<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+  <header class="top-nav">
+    <nav>
+      <a class="nav-link" href="services.html">Services</a>
+      <a class="nav-link" href="">Product</a>
+      <a class="nav-link" href="">Vision</a>
+      <a class="nav-link" href="">Features</a>
+      <a class="nav-link" href="">About</a>
+      <a class="nav-link" href="">Contact</a>
+    </nav>
+
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </header>
+
+<div class="container">
+
+  <section class="blurb-section">
+    <div class="blurb-text-div">
+      <h1 class="blurb-text">
+        Innovation<br/>
+        On<br/>
+        Demand<br/>
+      </h1>
+      <div class="blurb-button-div">
+        <button class="blurb-button button">Get Started</button>
+      </div>
+    </div>
+      <img class="blurb-img" src="img/header-img.png" alt="Image of a code snippet.">
+  </section>
+
+  <section class="section section1">
+    <section class="features section-content2by main-text">
+      <h2>Features</h2>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+
+    <section class="about section-content2by main-text">
+      <h2>About</h2>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+  </section>
+
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+  <section class="section section2">
+    <section class="section-content3by services main-text">
+      <h2>Services</h2>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+    <section class="section-content3by products main-text">
+      <h2>Product</h2>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+    <section class="section-content3by vision main-text">
+      <h2>Vision</h2>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+  </section>
+  <footer>
+    <h2>Contact</h2>
+    <address>
+      123 Way 456 Street<br/>
+      Somewhere, USA
+    </address>
+    <div class="phone">1 (888) 888-8888</div>
```

Comment:
```suggestion
    <a href="tel:18888888888">1 (888) 888-8888</a>
```
Putting this in an anchor tag will tell the browser that this is a phone number, similarly to how you have the email address in the line below 😄